### DISCUSSION TITLE
Suppress the unwanted errors

### DISCUSSION THREAD

**Discussion by shreesh52** [Ericsson/codechecker#4144] [General]
Hi all, I'm using CodeChecker, If I want to suppress the unwanted error largely like in some places; I wanted this for specifically for cppcoreguidelines-avoid-magic-numbers

**Comment by bruntib** [comment]
Hi,
There is a chapter in the user guide about the different possibilities of suppressing reports. A new feature has been introduced the in the latest release which makes it possible to suppress reports on filename or checker name granularity: https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#setting-with-config-file.
